examples/common-voice/scripts/prepare.bash

```diff
@@ -63,8 +64,9 @@ set -e
 for language in ${datasets[*]}; do
 	tarfile=$downloads_dir/${language}.tar.gz
 	echo "unpacking '$tarfile'"
-	mkdir --parents --verbose $output_dir/$language
+	mkdir -pv $output_dir/$language
```

Comment:
```suggestion
	mkdir --parents --verbose $output_dir/$language
```

Reply:
I don't think these are accepted options for `makedir` and `basename` anymore. Got this error during runtime (on OSX):
```
mkdir: illegal option -- -
usage: mkdir [-pv] [-m mode] directory ...
```

Reply:
Oh, I see, that's good to know. I've been running this script only on Linux. Let's go with the short format since it works with both.